docs/embedding/sdk/snippets/questions/interactive-question-click-actions.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  defineMetabaseAuthConfig,
+  InteractiveDashboard,
+  MetabaseProvider,
+} from "@metabase/embedding-sdk-react";
+
+const authConfig = defineMetabaseAuthConfig({
+  metabaseInstanceUrl: "http://localhost:3000",
+});
+
+const Example = () => {
+  return (
+    // [<snippet example>]
+    <MetabaseProvider
+      authConfig={authConfig}
+      pluginsConfig={{
+        mapQuestionClickActions: (clickActions, clicked) => {
+          if (clicked?.column?.display_name === "Last Name") {
+            // This adds a custom action to the menu when clicked on on "Last Name" column
+            return [
+              ...clickActions,
+              {
+                buttonType: "horizontal",
+                name: "custom",
+                title: "This is the Last Name column",
+                onClick: () => alert("You clicked the Last Name column!"),
+              },
+            ];
+          }
+
+          if (clicked?.column?.display_name === "Plan") {
+            // This performs an immediate action on "Plan" column insttad of opening the menu
```

Comment:
```suggestion
            // This performs an immediate action on "Plan" column instead of opening the menu
```
Though this will be overwritten...